editors/code/package.json

```diff
@@ -35,6 +35,29 @@
 				]
 			}
 		],
+		"configuration": {
+			"properties": {
+				"air.logLevel": {
+					"default": null,
+					"markdownDescription": "Controls the log level of the language server.\n\n**This setting requires a restart to take effect.**",
```

Comment:
These static settings are a bit annoying. Probably worth it to make it dynamic right away?